.github/dependabot.yml

```diff
@@ -11,4 +11,9 @@ updates:
   - package-ecosystem: "github-actions"
     directory: "/"
     schedule:
-      interval: "weekly"
\ No newline at end of file
+      interval: "weekly"
+
+  - package-ecosystem: npm
```

Comment:
There's already a block for the webview project (lines 7-10). It's identical to the new one but weekly instead of daily. All the other ones are weekly, so maybe we should keep this unchanged?

Reply:
Cool, post merge I can open PR to cater this. Discussed already.